lib/node_modules/@stdlib/string/base/replace-after-last/README.md

```diff
@@ -0,0 +1,122 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# replaceAfterLast
+
+> Replace the substring after the last occurrence of a specified search string.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var replaceAfterLast = require( '@stdlib/string/base/replace-after-last' );
+```
+
+#### replaceAfterLast( str, search, replacement )
```

Comment:
@AuenKr As mentioned in the OP of https://github.com/stdlib-js/stdlib/issues/814, the desire is to model similarly to `substring-after-last` which supports a `fromIndex` argument. It would be good to add support for that here.